### ISSUE TITLE
brew install pkg-config

### ISSUE THREAD

**Issue by zeneofa** [Homebrew/linuxbrew#101]
Hi,

I am having huge issues installing pkg-config using brew. On the system I am working on, I do not have root. When I run:

brew install pkg-config --verbose

I get the following errors: 
gconvert.c:66:2: error: #error GNU libiconv not in use but included iconv.h is from libiconv

brew doctor tells me:

Warning: libiconv files detected at a system prefix other than /usr
Homebrew doesn't provide a libiconv formula, and expects to link against
the system version in /usr. libiconv in other prefixes can cause
compile or link failure, especially if compiled with improper
architectures. OS X itself never installs anything to /usr/local so
it was either installed by a user or some other third party software.

tl;dr: delete these files:
    /usr/local/include/iconv.h

Now I have no way of removing this offending file, I do have iconv locally accessible:
~/local/bin/iconv

I suspect the above is caused by iconv.h being present and thus the system expects libiconv to be activated. I unfortunately cannot run ldconfig to activate libiconv. Is there some way to mask this iconv.h file or run a form of ldconfig locally or install pkg-config in some other way and then let brew detect that it is there?

Sorry I am a bit of noob with all this and this might not be a brew issue, but I had no clue where to ask this. 

(some background, the brew install pkg-config is a depedency resolution that gets called by bcbio-nextgen install script)
